libnopegl/src/pgcraft.c

```diff
@@ -523,10 +523,9 @@ static int inject_texture(struct pgcraft *s, struct pgcraft_texture *texture, st
 
 static int inject_textures(struct pgcraft *s, const struct pgcraft_params *params, int stage)
 {
-    struct darray *texture_infos_array = &s->texture_infos;
     struct pgcraft_texture *textures = ngli_darray_data(&s->textures);
-    struct pgcraft_texture_info *texture_infos = ngli_darray_data(texture_infos_array);
-    for (size_t i = 0; i < ngli_darray_count(texture_infos_array); i++) {
+    struct pgcraft_texture_info *texture_infos = ngli_darray_data(&s->texture_infos);
+    for (size_t i = 0; i < ngli_darray_count(&s->texture_infos); i++) {
```

Comment:
commit message is missing a `s` in "simplify inject_texture**s**()"

Reply:
Fixed.